tests/integration-tests/tests/iam/test_iam.py

```diff
@@ -45,7 +46,10 @@ def test_iam_roles(
     copyfile(config_file_path, updated_config_file_path)
 
     cluster_config = pcluster_config_reader(
-        config_file=config_file_name, ec2_iam_role=cluster_role_name, iam_lambda_role=lambda_role_name
+        config_file=config_file_name,
+        ec2_iam_role=cluster_role_name,
```

Comment:
we should rename cluster_role_name to cluster_role_arn or just cluster_role.
Same for lambda_role_name.

queue_instance_profile might just be instance_profile, in future we might use it for head node too.

Reply:
Renamed